dts/bindings/pinctrl/renesas,rcar-pfc.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2021 IoT.bzh
+# SPDX-License-Identifier: Apache-2.0
+
+description: Renesas R-Car Pin Function Controller node
```

Comment:
consider adding more detailed documentation, see for example nrf or gd32 bindings